### DISCUSSION TITLE
Virtiofs on Linux

### DISCUSSION THREAD

**Discussion by gehoern** [utmapp/UTM#4632] [UTM Help (macOS)] [answered]
**Describe the issue**

standard linux guest cannot mount virtiofs

**Configuration**
Apple M1
UTM 4.09
MacOS 13.0

**VMsetup**
...
-fsdev local,id=virtfs0,path=/Users/xyz/shared,security_model=mapped-xattr 
-device virtio-9p-pci,fsdev=virtfs0,mount_tag=share
...
default setup to share a folder via virtiofs 
automatically getting the tag "share" assigned
setting even access rights to world read and writable
  ```drwxrwxrwx   4 xyz  staff     128 Nov  8 22:06 shared```

**Guest usage**
```
root@debian:~# mount -t virtiofs share /mnt
mount: /mnt: wrong fs type, bad option, bad superblock on share, missing codepage or helper program, or other error.
       dmesg(1) may have more information after failed mount system call.
```

dmesg output:
```
[  284.004409] virtio-fs: tag <share> not found
```

what is wrong in this setup?
if i switch to "SPICE WebDAV" it immediately works.

**Comment by gehoern** [comment, marked as answer]
ok i guess i got pointed to the right solution:

Guest usage:
```mount -t 9p -o trans=virtio share [mount point] -oversion=9p2000.L```

and it works. my bad.

**Reply by lowrisk75** [reply]
it doesn't work for me I'm  having a similar issue but I don't get what I'm missing

**Comment by crazyn2** [comment]
I encountered the similar problem.
```
$ sudo mount -t virtiofs Ubuntus /home/kvm/Ubuntus
mount: /home/kvm/Ubuntus: wrong fs type, bad option, bad superblock on /home/kvm/Ubuntus, missing codepage or helper program, or other error.
```
I fixed it by changing its share name or name of the directory. The key point is that they can't be identical.

**Reply by rofrol** [reply]
Not true. They can be identical.
